pkg/bloomgateway/querier_test.go

```diff
@@ -55,8 +56,8 @@ func TestBloomQuerier(t *testing.T) {
 		through := model.Now()
 		from := through.Add(-12 * time.Hour)
 		chunkRefs := []*logproto.ChunkRef{}
-		filters := []*logproto.LineFilterExpression{
-			{Operator: 0, Match: "uuid"},
+		filters := []syntax.LineFilter{
+			{Ty: 0, Match: "uuid", Op: ""},
```

Comment:
Could you use the enums and constants from `syntax` instead of `0` and `""`?

Reply:
Op is usually "", unless there are nested OR filters

Reply:
I think I can omit `Op`